src/gcp_scanner/arguments.py

```diff
@@ -13,120 +13,135 @@
+    )
 
-  Returns:
-    argparse.Namespace: A namespace object containing the parsed command-line
-    arguments.
-  """
-  parser = argparse.ArgumentParser(
-      prog='scanner.py',
-      description='GCP Scanner',
-      usage='python3 %(prog)s -o folder_to_save_results -g -')
-  required_named = parser.add_argument_group('Required parameters')
-  required_named.add_argument(
-      '-o',
-      '--output-dir',
-      required=True,
-      dest='output',
-      default='scan_db',
-      help='Path to output directory')
+    parser.add_argument(
+        "-k",
+        "--sa-key-path",
+        default=None,
+        dest="key_path",
+        help="Path to directory with SA keys in json format",
+    )
+    parser.add_argument(
+        "-g",
+        "--gcloud-profile-path",
+        default=None,
+        dest="gcloud_profile_path",
+        help="Path to directory with gcloud profile. Specify -\
+ to search for credentials in default gcloud config path",
+    )
+    parser.add_argument(
+        "-m",
+        "--use-metadata",
+        default=False,
+        dest="use_metadata",
+        action="store_true",
+        help="Extract credentials from GCE instance metadata",
+    )
+    parser.add_argument(
+        "-at",
+        "--access-token-files",
+        default=None,
+        dest="access_token_files",
+        help="A list of comma separated files with access token and OAuth scopes.\
+TTL limited. A token and scopes should be stored in JSON format.",
+    )
+    parser.add_argument(
+        "-rt",
+        "--refresh-token-files",
+        default=None,
+        dest="refresh_token_files",
+        help="A list of comma separated files with refresh_token, client_id,\
+token_uri and client_secret stored in JSON format.",
+    )
+    parser.add_argument(
+        "-s",
+        "--service-account",
+        default=None,
+        dest="key_name",
+        help="Name of individual SA to scan",
+    )
+    parser.add_argument(
+        "-p",
+        "--project",
+        default=None,
+        dest="target_project",
+        help="Name of individual project to scan",
+    )
+    parser.add_argument(
+        "-f",
+        "--force-projects",
+        default=None,
+        dest="force_projects",
+        help="Comma separated list of project names to include in the scan",
+    )
+    parser.add_argument(
+        "-c",
+        "--config",
+        default=None,
+        dest="config_path",
+        help="A path to config file with a set of specific resources to scan.",
+    )
+    parser.add_argument(
+        "-l",
+        "--logging",
+        default="WARNING",
+        dest="log_level",
+        choices=("DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"),
+        help="Set logging level (INFO, WARNING, ERROR)",
+    )
+    parser.add_argument(
+        "-lf",
+        "--log-file",
+        default=None,
+        dest="log_file",
+        help="Save logs to the path specified rather than displaying in\
+ console",
+    )
```

Comment:
Could you also try removing the unnecessary whitespaces and indentations, these create a great amount of confusion when reviewing the file and some changes could go unnoticed.

Thanks!